### DISCUSSION TITLE
Is there any method to switch the typewriting while changing the Insert mode with visual mode

### DISCUSSION THREAD

**Discussion by ltx0621** [esm7/obsidian-vimrc-support#34] [Q&A]
As a non-English user,Writing another language on vim editor is useful in my work and life.But must switch the typewriting after changing to Visual mode from Insert mode is a terrible experience.
Thank you for your time

**Comment by esm7** [comment]
I suffer from this too and already spent some time looking for a solution, but wasn't able to find a reasonable one yet.
Good to know I'm not the only one, it helps my incentive to keep looking :)
